doc/admin-guide/files/records.config.en.rst

```diff
@@ -1347,6 +1347,32 @@ Parent Proxy Configuration
    ``2`` Mark the host down. This is the default.
    ===== ======================================================================
 
+.. ts:cv:: CONFIG proxy.config.http.parent_proxy.enable_parent_timeout_markdowns INT 0
+   :reloadable:
+   :overridable:
+
+   Enables (``1``) or disables (``0``) parent proxy mark downs due to inactivity
+   timeouts.  By default parent proxies are not marked down due to inactivity
+   timeouts, the transaction will retry using another parent instead.  The
+   default for this configuration keeps this behavior and is disabled (``0``).
+   This setting is overridable using one of the two plugins ``header_rewrite``
+   or ``conf_remap`` to enable inactivity timeout markdowns and should be done
```

Comment:
Why do you feel like we should mention these particular plugins as the ones overriding the configs?  Any plugin that uses these TS API calls can override them:  https://github.com/apache/trafficserver/blob/4e6dc02f7ef7ae15ebda6c3bdf54e80f1ec55ba5/include/ts/ts.h#L2459 .  The lua plugin could also override them.